From 6b992920bfde1a7aff69eaad6918c009a27b98fe Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 4 Sep 2020 09:27:47 +0000
Subject: [PATCH] IWYU: add missing bitset include in AXEventGenerator

---
 ui/accessibility/ax_event_generator.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/accessibility/ax_event_generator.h b/ui/accessibility/ax_event_generator.h
index 5fa2439..85f3209 100644
--- a/ui/accessibility/ax_event_generator.h
+++ b/ui/accessibility/ax_event_generator.h
@@ -5,6 +5,7 @@
 #ifndef UI_ACCESSIBILITY_AX_EVENT_GENERATOR_H_
 #define UI_ACCESSIBILITY_AX_EVENT_GENERATOR_H_
 
+#include <bitset>
 #include <map>
 #include <ostream>
 #include <set>
-- 
2.26.2

