From 49c9ab14152b52edc9f0d8f1245039792f270315 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 7 May 2021 09:53:18 +0000
Subject: [PATCH] IWYU: include atomic for std::atomic

---
 base/threading/platform_thread.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/threading/platform_thread.cc b/base/threading/platform_thread.cc
index 3dc2181..f167af4 100644
--- a/base/threading/platform_thread.cc
+++ b/base/threading/platform_thread.cc
@@ -4,6 +4,7 @@
 
 #include "base/threading/platform_thread.h"
 
+#include <atomic>
 #include <memory>
 
 #include "base/feature_list.h"
-- 
2.26.3

