From b6eb6013cd7bebc548677195a8fbd5a48644fe99 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 26 Sep 2021 07:32:26 +0000
Subject: [PATCH] IWYU: add vector for std::vector in messaging_util

---
 extensions/renderer/messaging_util.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extensions/renderer/messaging_util.h b/extensions/renderer/messaging_util.h
index 8c30d8d..525ff1d 100644
--- a/extensions/renderer/messaging_util.h
+++ b/extensions/renderer/messaging_util.h
@@ -7,6 +7,7 @@
 
 #include <memory>
 #include <string>
+#include <vector>
 
 #include "v8/include/v8-forward.h"
 
-- 
2.32.0

