From a75b7265e7429aa48f180dcaf9652a1f3c722718 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 10 Jan 2021 08:13:31 +0000
Subject: [PATCH] IWYU: add missing include for size_t

---
 third_party/dawn/src/common/Alloc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/dawn/src/common/Alloc.h b/third_party/dawn/src/common/Alloc.h
index 5a9fb08..940d5ff 100644
--- a/third_party/dawn/src/common/Alloc.h
+++ b/third_party/dawn/src/common/Alloc.h
@@ -15,6 +15,7 @@
 #ifndef COMMON_ALLOC_H_
 #define COMMON_ALLOC_H_
 
+#include <cstddef>
 #include <new>
 
 template <typename T>
-- 
2.26.2

