From 74608652d7ec89b8254a699da5b13b529890b5f2 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 16 Dec 2020 16:15:12 +0000
Subject: [PATCH] IWYU: add missing include for std::unique_ptr

---
 ui/webui/webui_config.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/webui/webui_config.h b/ui/webui/webui_config.h
index 29990bd..9664e1f 100644
--- a/ui/webui/webui_config.h
+++ b/ui/webui/webui_config.h
@@ -5,6 +5,7 @@
 #ifndef UI_WEBUI_WEBUI_CONFIG_H_
 #define UI_WEBUI_WEBUI_CONFIG_H_
 
+#include <memory>
 #include <string>
 
 #include "base/strings/string_piece.h"
-- 
2.26.2

