From 46503cdc68c76d776e204869da3c503a7d8aaa0a Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 8 Feb 2021 11:39:22 +0000
Subject: [PATCH] IWYU: add missing include for std::unique_ptr

---
 ui/gfx/rendering_stage_scheduler.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gfx/rendering_stage_scheduler.h b/ui/gfx/rendering_stage_scheduler.h
index 701f62b..3c00f17 100644
--- a/ui/gfx/rendering_stage_scheduler.h
+++ b/ui/gfx/rendering_stage_scheduler.h
@@ -5,6 +5,7 @@
 #ifndef UI_GFX_RENDERING_STAGE_SCHEDULER_H_
 #define UI_GFX_RENDERING_STAGE_SCHEDULER_H_
 
+#include <memory>
 #include <vector>
 
 #include "base/threading/platform_thread.h"
-- 
2.26.2

