From 444330b6e7f23e7116b2db7f748ea7b14f4ae4ee Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 4 Dec 2021 08:20:26 +0000
Subject: [PATCH] IWYU: add limits for std::numeric_limits in gfx::CubicBezier

---
 ui/gfx/geometry/cubic_bezier.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gfx/geometry/cubic_bezier.cc b/ui/gfx/geometry/cubic_bezier.cc
index 8546aec..3064548 100644
--- a/ui/gfx/geometry/cubic_bezier.cc
+++ b/ui/gfx/geometry/cubic_bezier.cc
@@ -6,6 +6,7 @@
 
 #include <algorithm>
 #include <cmath>
+#include <limits>
 
 #include "base/check_op.h"
 #include "base/cxx17_backports.h"
-- 
2.32.0

