(taken from c++17 reland^2)
--- a/content/public/browser/web_contents_user_data.h
+++ b/content/public/browser/web_contents_user_data.h
@@ -91,7 +91,7 @@ class WebContentsUserData : public base::SupportsUserData::Data {
 // This macro declares a static variable inside the class that inherits from
 // WebContentsUserData The address of this static variable is used as the key to
 // store/retrieve an instance of the class on/from a WebState.
-#define WEB_CONTENTS_USER_DATA_KEY_DECL() static constexpr int kUserDataKey = 0
+#define WEB_CONTENTS_USER_DATA_KEY_DECL() static const int kUserDataKey = 0
 
 // This macro instantiates the static variable declared by the previous macro.
 // It must live in a .cc file to ensure that there is only one instantiation
