From 01919473cf683447b5a7c638a44528e8e80a6f3a Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 21 Apr 2021 16:53:19 +0000
Subject: [PATCH] IWYU: include memory for std::unique_ptr

---
 components/power_scheduler/power_mode_voter.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/components/power_scheduler/power_mode_voter.h b/components/power_scheduler/power_mode_voter.h
index f6bf513..d375ec9 100644
--- a/components/power_scheduler/power_mode_voter.h
+++ b/components/power_scheduler/power_mode_voter.h
@@ -5,6 +5,8 @@
 #ifndef COMPONENTS_POWER_SCHEDULER_POWER_MODE_VOTER_H_
 #define COMPONENTS_POWER_SCHEDULER_POWER_MODE_VOTER_H_
 
+#include <memory>
+
 #include "base/component_export.h"
 #include "base/time/time.h"
 #include "components/power_scheduler/power_mode.h"
-- 
2.26.3

