From 10976f5ffdb5f74c165abfd9d585896bfdc540d1 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 28 Nov 2020 20:22:27 +0000
Subject: [PATCH] IWYU: add missing include for std::condition_variable

---
 base/allocator/partition_allocator/pcscan.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/allocator/partition_allocator/pcscan.cc b/base/allocator/partition_allocator/pcscan.cc
index 49ccdc0..4b59757 100644
--- a/base/allocator/partition_allocator/pcscan.cc
+++ b/base/allocator/partition_allocator/pcscan.cc
@@ -5,6 +5,7 @@
 #include "base/allocator/partition_allocator/pcscan.h"
 
 #include <algorithm>
+#include <condition_variable>
 #include <map>
 #include <mutex>
 #include <numeric>
-- 
2.26.2

