From 899e0f1e4007a9067f3629811c509f9e6a24b406 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Wed, 5 Aug 2020 08:10:16 +0000
Subject: [PATCH] IWYU: std::vector usage in fuzzy_finder.h requires <vector>

---
 chrome/browser/ui/commander/fuzzy_finder.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/chrome/browser/ui/commander/fuzzy_finder.h b/chrome/browser/ui/commander/fuzzy_finder.h
index 708b981..4b01797 100644
--- a/chrome/browser/ui/commander/fuzzy_finder.h
+++ b/chrome/browser/ui/commander/fuzzy_finder.h
@@ -5,6 +5,8 @@
 #ifndef CHROME_BROWSER_UI_COMMANDER_FUZZY_FINDER_H_
 #define CHROME_BROWSER_UI_COMMANDER_FUZZY_FINDER_H_
 
+#include <vector>
+
 #include "base/strings/string16.h"
 #include "ui/gfx/range/range.h"
 
-- 
2.26.2

